packages/storage/src/service.ts

```diff
@@ -88,145 +179,50 @@ export class StorageService {
 
   /**
    * Stop running requests and prevent more from being created.
+   * @internal
    */
   deleteApp(): void {
```

Comment:
Is it the function that executes when calling `app.delete()`? If so, it should be renamed to `_delete()` following the [interface](https://github.com/firebase/firebase-js-sdk/blob/master/packages-exp/app-types-exp/index.d.ts#L110).

Reply:
Yes, done.